src/contract.rs

```diff
@@ -73,9 +76,120 @@ pub fn execute(
             validator_oper_addr,
         } => remove_validator(deps, info, validator_oper_addr),
         ExecuteMsg::WithdrawFunds { amount } => withdraw_funds(deps, info, amount),
+
```

Comment:
Can you also add a migrate message to this contract with the msg taking in a manager address to be updated?
This will help us with replacing the code in prod with the new code you would have authored.

Reply:
Need some help in understanding how to do the migration, and what all it entails. I am assuming the following is required:
1. Migrate the code id of the contract to new code.
2. Have a way to migrate any new ds changes to be consistent.

I am assuming no. 2 will have significant gas costs as well?

Reply:
I wouldn't recommend DS changes as part of migration. There is a way to do it but for your changes, I wouldn't want to invest in that.

Reply:
https://github.com/stader-labs/stader-protocol-v1/blob/cryptonics_commit/contracts/pools/src/contract.rs#L68
Add something similar to this.